lib/http.js

```diff
@@ -748,6 +750,69 @@ async function _generateRootCapability(url) {
   };
 }
 
+/**
+ * Gets the expectedRootCapability for a zCap from the invocationTarget.
+ *
+ * @param {object} Options to use.
+ * @param {string|Array<string>} options.invocationTarget - The
+ *   invocationTarget.
+ * @param {string} options.edvId - The edvId.
+ * @param {string} [options.docPath] - An optional docPath.
+ *
+ * @return {string|Array<string>} The expectedRootCapability for
+ *   the invocationTarget.
+ */
+function _getExpectedRootCapability({invocationTarget = '', edvId, docPath}) {
+  const verifyDocPath = url =>
+    url.includes('documents') && !url.endsWith('documents');
```

Comment:
We should move this out to a separate helper function so we don't create a new function each time this is called.

Reply:
Also, based on the code path below, once the `invocationTarget` is an array path is removed, we won't need to check for `url.endsWith(documents)` anymore. It seems like we can just check to see if the target ends with `/documents/${docPath}` ... though, looking at where this gets called, `docPath` is never passed. Seems like there's more to do here.